debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -174,15 +174,16 @@ public SqlServerConnection(SqlServerJdbcConfiguration config, SqlServerValueConv
     /**
      * Creates a new connection using the supplied configuration.
      *
-     * @param config              {@link Configuration} instance, may not be null.
+     * @param connectorConfig              {@link Configuration} instance, may not be null.
+     * @param jdbcConfig                   {@link SqlServerJdbcConfiguration} instance, may not be null.
      * @param valueConverters     {@link SqlServerValueConverters} instance
      * @param skippedOperations   a set of {@link Envelope.Operation} to skip in streaming
      * @param optionRecompile     Includes query option RECOMPILE on incremental snapshots
      */
-    public SqlServerConnection(SqlServerJdbcConfiguration config, SqlServerValueConverters valueConverters,
+    public SqlServerConnection(SqlServerJdbcConfiguration jdbcConfig, SqlServerConnectorConfig connectorConfig, SqlServerValueConverters valueConverters,
```

Comment:
@ani-sha 
I think it is not necessary to pass another parameter. Just change `config` to `SqlServerConnectorConfig` and then call `config.getJdbcConfig()` to obtain JDBC configuration.

Reply:
I did that at first, but how to handle `TestHelper.adminConnection`and `TestHelper.testConnection` which pass `JdbcConfig`?

Reply:
Would you be able to create `SqlServerConnectorConfig` in those tow methods?